util/os-resources.py

```diff
@@ -0,0 +1,84 @@
+# !/usr/bin/python3
+import click
+import shutil
+import os
+
+@click.group()
+def cli():
+    pass
+
+@cli.command()
+@click.option("--cookbooks-path", help="Directory containing the cookbooks.", required=True, type=click.Path())
+@click.option("--src-platform", help="Platform of the os to copy from (e.g. redhat)", required=True)
```

Comment:
minor: we know the available source os, so we could specify the valid values, for src- options.

Reply:
added.